src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -113,7 +113,8 @@ public SscsCaseData getOldSscsCaseData() {
     private boolean hasAppointee() {
         return (responseWrapper.getNewSscsCaseData().getAppeal() != null
             && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant() != null
-            && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee() != null);
+            && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee() != null
+            && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee().getName().getLastName() != null);
```

Comment:
Why does this check for the appointee last name? Should it not be checking to see if getName() is != null?

Reply:
Basically getName() is not null but the values of individual properties are null i.e. Appointee(name=Name(title=null, firstName=null, lastName=null)